druid-shell/src/platform/mac/window.rs

```diff
@@ -947,18 +947,14 @@ impl WindowHandle {
     }
 
     pub fn open_file(&mut self, options: FileDialogOptions) -> Option<FileDialogToken> {
-        self.open_save_impl(FileDialogType::Open, options)
+        Some(self.open_save_impl(FileDialogType::Open, options))
```

Comment:
It looks like you ran clippy on mac. I get similar warnings on Windows in `vk_to_key` and `key_to_vk` (druid-shell/src/platform/windows/keyboard.rs), and `init` in platform/windows/application.rs. Wouldn't be surprised if there was a similar story on Linux.

Reply:
ah good call.

Reply:
that looks to be a slightly different problem which I'd prefer to diagnose when I have a windows machine in front of me, since it also isn't showing up in ci. :/